### PR TITLE
Feature/VOTE-2534 Enable Japanese translation

### PR DESCRIPTION
**Pull request by daniel-crowder**
## Jira ticket

[VOTE-2142](https://cm-jira.usa.gov/browse/VOTE-2142)

## Description

Add PO and Config files to enable and provide string translations for Japanese.

### Pre-deploy

n/a

### Post-deploy

1. Locally, run `lando retune -y` to import config files and string translations from PO file.

### QA/Test

1.  Visit http://vote-gov.lndo.site/admin/config/regional/language and confirm that Japanese language is listed.
2. Visit http://vote-gov.lndo.site/admin/config/regional/translate
3. Do a search using the following: Translation: Japanese, Search In: Only translated strings
4. Confirm that the search returns a list of translated strings for Japanese.

## Checklist for the Developer

- [x] A link to the JIRA ticket has been included above.
- [x] No merge conflicts exist with the target branch.
- [x] Automated tests have passed on this PR.
- [x] A reviewer has been designated.
- [x] Deployment and testing steps have been documented above, if applicable.

## Checklist for the Peer Reviewers

- [ ] The file changes are relevant to the task objective.
- [ ] Code is readable and includes appropriate commenting.
- [ ] Code standards and best practices are followed.
- [ ] QA/Test steps were successfully completed, if applicable.
- [ ] Applicable logs are free of errors.

### CODE REVIEW

File: config/sync/language/ja/core.date_format.revision_date.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+pattern: 'Y年F月d日'
```
Comment (rayestrada) [inline config/sync/language/ja/core.date_format.revision_date.yml:1]: I would remove this character `月` if you doing so in the string translation.

File: config/sync/language/ja/core.date_format.long_date.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+pattern: 'Y年F月j日l'
```
Comment (rayestrada) [inline config/sync/language/ja/core.date_format.long_date.yml:1]: I would remove this character `月` if you doing so in the string translation.

**Review by rayestrada — CHANGES_REQUESTED** [review]

File: config/sync/language/ja/core.date_format.long_date.yml (hunk 2 above)
Comment (daniel-crowder) [reply config/sync/language/ja/core.date_format.long_date.yml]: Updated!

File: config/sync/language/ja/core.date_format.revision_date.yml (hunk 1 above)
Comment (daniel-crowder) [reply config/sync/language/ja/core.date_format.revision_date.yml]: Updated!

**Review by rayestrada — APPROVED** [review]

**Pull request merged**